values.yaml

```diff
@@ -365,7 +366,7 @@ jobservice:
 registry:
   registry:
     image:
-      repository: goharbor/registry-photon
+      repository: registry-photon
```

Comment:
why not just let user set the repo with registry's url?

Reply:
we have so many repos, user don't need to set each one, and that would make the wrapper scripts too complex.
My target is just push the harbor images to some different registries.

Reply:
I don't think we should trade flexibility for a shortcut.   
For example, when I'm debugging the chart I can push an image to my local registry or my personal namespace and use official images for other pods.

If you don't like editing the values.yaml, you can write a script or use tools like kustomize to patch the yaml?